gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -439,6 +441,45 @@ gum_stalker_finalize (GObject * object)
   G_OBJECT_CLASS (gum_stalker_parent_class)->finalize (object);
 }
 
+static GumVirtualizationRequirements
+gum_exec_block_virtualize_linux_sysenter_insn (GumExecBlock * block,
+                                               GumGeneratorContext * gc)
+{
+  GumArm64Writer *cw = gc->code_writer;
+  gconstpointer continue_normally = cw->code + 1;
+  gconstpointer end_payload = cw->code + 3;
```

Comment:
```suggestion
  gconstpointer end_payload = cw->code + 2;
```

(These just have to be unique, but should be +1 relative to each order for readability.)

Reply:
updated